client/my-sites/upgrades/plans-compare/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getPlans } from 'state/plans/selectors';
+import { getPlansBySite } from 'state/sites/plans/selectors';
+import PlansCompare from 'components/plans/plans-compare' ;
+
+const UpgradePlansCompare = ( { domain, sites, plans, features, productsList } ) => {
+	const selectedSite = sites.getSelectedSite();
+	const backUrl = '/domains/add/' + domain + '/plans/' + selectedSite.slug;
+
+	return (
+		<div className="plans has-sidebar">
+			<PlansCompare { ...{ selectedSite, plans, features, productsList, backUrl } }
+				hideFreePlan={ true }
+				isInSignup={ true } />
+		</div>
+	);
+};
+
+UpgradePlansCompare.propTypes = {
+	domain: React.PropTypes.string.isRequired,
+	sites: React.PropTypes.object.isRequired,
+	plans: React.PropTypes.array.isRequired,
+	features: React.PropTypes.object.isRequired,
+	productsList: React.PropTypes.object.isRequired
+};
+
+export default connect(
+	( state, props ) => {
+		return {
+			plans: getPlans( state ),
+			sitePlans: getPlansBySite( state, props.sites.getSelectedSite )
+		};
+	}
```

Comment:
the explicit `return` here seems redundant. we could just use the fat-arrow syntax to return an object directly

```js
( state, props ) => ( {
	plans: getPlans( state ),
	sitePlans: getPlansBySite( state, props.sites.getSelectedSite )
} )
```